ocs_ci/utility/kms.py

```diff
@@ -178,6 +179,26 @@ def vault_namespace_exists(self, vault_namespace):
                 return False
         return True
 
+    def vault_backend_path_exists(self, backend_path):
+        """
+        Check if vault backend path already exists
+
+        Args:
+            backend_path (str): name of the vault backend path
+
+        Returns:
+            bool: True if exists else False
+
+        """
+        cmd = f"vault secrets list | grep {backend_path}"
+        proc = subprocess.Popen(
```

Comment:
I think you can use ocs_ci.utility.utils.exec_cmd which will return `rt, out, err`